sys-boot/gnu-efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,29 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), \
+and IA-32 (x86) platforms using the GNU toolchain and the EFI development \
+environment."
+HOMEPAGE="https://sourceforge.net/projects/gnu-efi/"
+COPYRIGHT="1999-2007 Hewlett-Packard Co.
+	2006-2010 Intel Co."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://downloads.sf.net/gnu-efi/gnu-efi-$portVersion.tar.bz2"
+CHECKSUM_SHA256="51a00428c3ccb96db24089ed8394843c4f83cf8f42c6a4dfddb4b7c23f2bf8af"
+SOURCE_DIR="gnu-efi-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
@ohnx, @korli, @jessicah, both Gentoo and ArchLinux ship libefi.a and libgnuefi.a, so I guess we should replace
```
ARCHITECTURES="any"
DISABLE_SOURCE_PACKAGE="yes"
```
by
```
ARCHITECTURES="x86_gcc2 x86 x86_64"
```
or
```
ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
```
and call make in lib in order to build these static libs too.

CCing @tqh, as I guess he is an expert on the UEFI topic ;-)

@ohnx, you forgot to click Submit on the GCI site.
I did not wait for it, however, and have already started investigating the topic since a few days but did not come to anything.
I noticed that your recipe is installing a Makefile that has paths which don't exist on Haiku. Should'nt we _use_ these Makefiles (to build the static libs) instead of copying them to the package?